Solution1/WeatherApi/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
what is the sense of this duplication in .Development?

Reply:
+